crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -9,7 +9,7 @@ use serde::{ser::SerializeMap, Deserialize, Serialize, Serializer};
 
 /// And 8-byte identifier for an execution payload.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize, Hash)]
-pub struct PayloadId(H64);
+pub struct PayloadId(pub H64);
```

Comment:
please leave this private
we shouldn't need this after

https://github.com/paradigmxyz/reth/pull/2149